llvm/lib/Transforms/Yk/ShadowStack.cpp

```diff
@@ -146,7 +146,7 @@ class YkShadowStack : public ModulePass {
             // Calculate this `AllocaInst`s size, aligning its pointer if
             // necessary, and create a replacement pointer into the shadow
             // stack.
-            size_t AllocaSize = *AllocaSizeInBits / 8;
+            size_t AllocaSize = *AllocaSizeInBits / sizeof(size_t);
```

Comment:
`size_t` probably isn't the right size here (unless that's what's commonly used elsewhere in LLVM?). Exactly what *is* the right size is a bit hard to know, but `uintptr_t` might be better?

Reply:
Agreed. Fixed in [e19be2d](https://github.com/ykjit/ykllvm/pull/53/commits/e19be2dc52f4f87f0be281f9e91031dc5c715cde)